src/realm/array.hpp

```diff
@@ -556,35 +455,17 @@ class Array : public ArrayParent {
 
     bool minimum(int64_t& result, size_t start = 0, size_t end = size_t(-1), size_t* return_ndx = nullptr) const;
 
-    /// This information is guaranteed to be cached in the array accessor.
-    bool is_inner_bptree_node() const noexcept;
-
-    /// Returns true if type is either type_HasRefs or type_InnerColumnNode.
-    ///
-    /// This information is guaranteed to be cached in the array accessor.
-    bool has_refs() const noexcept;
-
-    /// This information is guaranteed to be cached in the array accessor.
-    ///
-    /// Columns and indexes can use the context bit to differentiate leaf types.
-    bool get_context_flag() const noexcept;
-    void set_context_flag(bool) noexcept;
-
-    ref_type get_ref() const noexcept;
-    MemRef get_mem() const noexcept;
-
     /// Destroy only the array that this accessor is attached to, not the
     /// children of that array. See non-static destroy_deep() for an
     /// alternative. If this accessor is already in the detached state, this
     /// function has no effect (idempotency).
-    void destroy() noexcept;
-
+    using DbElement::destroy;
```

Comment:
Do these using statements "reexport" the `DbElement` methods as `Array` methods?